src/map/attack.cpp

```diff
@@ -346,6 +351,11 @@ bool CAttack::IsCountered()
 
 bool CAttack::CheckCounter()
 {
+    if (!m_victim->PAI->IsEngaged())
+    {
+        return m_isCountered;
```

Comment:
i know that m_isCountered is initialized to false.. but can we set it to false again before returning here, just in case someone doesn't change how it works later?